RestoreVMFromRestorePoint.json

```diff
@@ -0,0 +1,249 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "<location>"
+        },
+        "networkInterfaceName": {
+            "type": "string",
+            "defaultValue": "<nicName>"
+        },
+        "networkSecurityGroupName": {
+            "type": "string",
+            "defaultValue": "<nsgName>"
+        },
+        "subnetName": {
+            "type": "string",
+            "defaultValue": "<subnetName>"
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "<vnetName>"
+        },
+        "publicIpAddressName": {
+            "type": "string",
+            "defaultValue": "<publicIPName>"
+        },
+        "virtualMachineName": {
+            "type": "string",
+            "defaultValue": "<vmName>"
+        },
+        "virtualMachineComputerName": {
+            "type": "string",
+            "defaultValue": "<computeName>"
+        },
+        "virtualMachineSize": {
+            "type": "string",
+            "defaultValue": "<vmSize>"
+        },
+        "adminUsername": {
```

Comment:
Shall we also add osProfile to use these?

Reply:
osProfile is not allowed for "Attach OSDisk" scenario, since the OSDisk is already specialized.